Makefile

```diff
@@ -0,0 +1,27 @@
+VERSION?=$(shell cat VERSION)
+IMAGE_TAG?=v$(VERSION)
+IMAGE_PREFIX?=registry.cn-hangzhou.aliyuncs.com/appcenter
+DOCKER_PUSH?=true
+
+.PHONY: build
+build:
+	CGO_ENABLED=0 go build -ldflags="-w -s" -o ./dist/argocd-aplicationset .
+
+.PHONY: image
+image:
+	docker build -t registry.cn-hangzhou.aliyuncs.com/appcenter/argocd-aplicationset:$(IMAGE_TAG) .
```

Comment:
Could we make the registry configurable as well, $IMAGE perhaps?

Reply:
I agree. Can make it such that it builds a local image (without a registry), but will prepend `IMAGE_PREFIX` if it is defined?

Reply:
Done